service/src/main/resources/application.yaml

```diff
@@ -59,3 +59,50 @@ fpl:
     mapping: 'example=>cafcass:cafcass@cafcass.com'
   local_authority_user:
     mapping: 'example=>1,2,3'
+
+---
+
+spring:
+  profiles: local
+
+
+management:
+  endpoints:
+    enabled-by-default: false
```

Comment:
Will that disable health check endpoint? I can imagine having management endpoint might be helpful locally sometimes. I suggest removing it that will result in all endpoints be enabled. We just disabled these endpoints in integration tests where they serve no purpose.

Reply:
That's true, I followed what we have got in our docker. I will readable it.